app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -1771,8 +1770,17 @@ private void showPlaybackProgress(final long progress, final long duration) {
         final int progressSeconds = (int) TimeUnit.MILLISECONDS.toSeconds(progress);
         final int durationSeconds = (int) TimeUnit.MILLISECONDS.toSeconds(duration);
         positionView.setMax(durationSeconds);
-        positionView.setProgressAnimated(progressSeconds);
-        detailPositionView.setText(Localization.getDurationString(progressSeconds));
+        // If there is no player inside fragment use animation, otherwise don't because
```

Comment:
So I took a look, but I don't understand in what case there would be no player in the fragment. I also think we shouldn't update the progress in the player when the controls aren't being shown.

Reply:
Start playing video in fragment, press popup button, voilà - player is not in fragment and this progress bar still shows current position.
Another situation: play video, close it via notification. Same thing.
Same with background playback. So this red progress bar at the bottom of thumbnail and it should be in sync with playback position from player.

Reply:
Ah, I didn't think of that, but why would we still want an animation in those cases?

Reply:
It is how it worked before me. And I think it looks cool. Open a video that you watched before and stopped on 50% of total duration. You'll see an animation. Isn't it cool?
Why do you want to remove it?

Reply:
Ah, yes, that's cool, but wom't this mean the additional CPU usage will still be there for e.g. the background player?

Reply:
Made better logic in this place. Now it's smart:)